README.md

```diff
@@ -100,6 +100,38 @@ lock(resource: 'some_resource', skipIfLocked: true) {
 }
 ```
 
+#### Finding Examples
+
+*List all resources acquired by the current build*
+
+```groovy
+lock(label:'printer') {
+  echo findLocks().name
+}
+```
+
+*List all resources (locked or not) currently defined*
+
+```groovy
+echo findLocks(build:'any').name
+```
+
+*List all resources (locked or not) with label `printer` excluding those with label `offline`*
+
+```groovy
+echo findLocks(anyOfLabels:'printer', noneOfLabels:'offline', build:'any').name
```

Comment:
How would this behave when `findLocks()` returns several values? Would Groovy resolve the `.name` part nicely, for each entry?

Reply:
yes, the magic of groovy - if you `.name` on an array of object, it applies the `.name` on each entry and returns an array of the results